pkg/mcp/configz/configz_test.go

```diff
@@ -66,9 +66,13 @@ func TestConfigZ(t *testing.T) {
 	defer cancel()
 
 	o := ctrlz.DefaultOptions()
-	ctrlz, _ := ctrlz.Run(o, []fw.Topic{CreateTopic(cl)})
+	o.PickPort = true
```

Comment:
I think we should pick the port in the test itself. There is some prior art for this that probes ports and finds an empty one.

Reply:
It is wonky to use a random port in the production case. Can we at least hide the server picked port, so that it doesn't accidentally make its way in to production code?

Reply:
+1 for letting the OS (e.g. `net.Listen`) pick the port.

Reply:
ozevren@ do you mean to unexport PickPort? That would limit it to ctrlz_test, but configz_test also needs that. But I have trouble imagining how can somebody accidentally set PickPort to true and not expect what will happen?

Reply:
That somebody could still do o.Port = PickUnusedPort() if they really wanted to, except in that case we get racy code. It takes 3 lines to implement PickUnusedPort so I wouldn't rely on laziness to prevent it either. It seems that either it won't matter (because nobody will care to use the feature) or we will encourage writing poor code (by preventing them from letting the system pick a port)? Seems whoever runs this gets to choose the port already, so it's not like we try to rely on the port having a fixed value?

Reply:
PTAL. As discussed offline, added an explicit Enabled option (DefaultOptions sets it to true), 0 port not auto picks (flag behavior is retained, ie. 0 port sets Enabled to false). You can inspect the actual address with Addr method.